sdk/loadtestservice/Azure.Developer.LoadTesting/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.2 (Unreleased)
```

Comment:
Just a head-up. If you want to release `1.0.2`, replace `Unreleased` with a release date in future.

Reply:
yes, got you, will create release for all the SDKs together in a separate PR